examples/schelling/model.py

```diff
@@ -95,5 +88,4 @@ def step(self):
 
         self.datacollector.collect(self)
 
-        if self.happy == len(self.agents):
-            self.running = False
+        self.running = self.happy != len(self.agents)
```

Comment:
While this is fully correct, I find it less readable. Maybe this could also be resolved with a comment.

Reply:
My experience is the other way around. I found this very clear.